code/src/main/java/com/onlinecontacttracing/storage/data-storage/PotentialContact.java

```diff
@@ -21,6 +24,31 @@ public String getEmail() {
     return emailOfPerson;
   }
 
+  /*
+  *  I added equals and hashCode so that we can use a hashSet when adding PotentialContacts
```

Comment:
Nice, but remove the "I added", instead just saying "equals and hashcode are defined so that... "

Reply:
Also maybe note here that we treat any two people with the same email address as the same user.

Reply:
sure thing